inc/appliance.class.php

```diff
@@ -0,0 +1,436 @@
+         ->addStandardTab('Change_Item', $ong, $options)
+         ->addStandardTab('Notepad', $ong, $options)
+         ->addStandardTab('Log', $ong, $options);
+
+      return $ong;
+   }
+
+
+   function showForm($ID, $options = []) {
+      $rand = mt_rand();
+
+      $this->initForm($ID, $options);
+      $this->showFormHeader($options);
+
+      echo "<tr class='tab_bg_1'>";
+
+      $rand = mt_rand();
+      $tplmark = $this->getAutofillMark('name', $options);
+
+      //TRANS: %1$s is a string, %2$s a second one without spaces between them : to change for RTL
+      echo "<td><label for='textfield_name$rand'>".sprintf(__('%1$s%2$s'), __('Name'), $tplmark) .
+           "</label></td>";
+      echo "<td>";
+      $objectName = autoName($this->fields["name"], "name",
+                             (isset($options['withtemplate']) && ( $options['withtemplate']== 2)),
+                             $this->getType(), $this->fields["entities_id"]);
+      Html::autocompletionTextField(
+         $this,
+         'name',
+         [
+            'value'     => $objectName,
+            'rand'      => $rand
+         ]
+      );
+      echo "</td>";
+      $randDropdown = mt_rand();
+      echo "<td><label for='dropdown_states_id$randDropdown'>".__('Status')."</label></td>";
+      echo "<td>";
+      State::dropdown([
+         'value'     => $this->fields["states_id"],
+         'entity'    => $this->fields["entities_id"],
+         'condition' => ['is_visible_appliance' => 1],
+         'rand'      => $randDropdown
+      ]);
+      echo "</td></tr>\n";
+
+      echo "<tr class='tab_bg_1'>";
+      $randDropdown = mt_rand();
+      echo "<td><label for='dropdown_locations_id$randDropdown'>".__('Location')."</label></td>";
+      echo "<td>";
+      Location::dropdown(['value'  => $this->fields["locations_id"],
+                               'entity' => $this->fields["entities_id"],
+                               'rand' => $randDropdown]);
+      echo "</td>";
+      $randDropdown = mt_rand();
+      echo "<td><label for='dropdown_appliancetypes_id$randDropdown'>".__('Type')."</label></td>";
+      echo "<td>";
+      ApplianceType::dropdown(['value' => $this->fields["appliancetypes_id"], 'rand' => $randDropdown]);
+      echo "</td></tr>\n";
+
+      echo "<tr class='tab_bg_1'>";
+      $randDropdown = mt_rand();
+      echo "<td><label for='dropdown_users_id_tech$randDropdown'>".__('Technician in charge of the appliance')."</label></td>";
+      echo "<td>";
+      User::dropdown(['name'   => 'users_id_tech',
+                           'value'  => $this->fields["users_id_tech"],
+                           'right'  => 'own_ticket',
+                           'entity' => $this->fields["entities_id"],
+                           'rand'   => $randDropdown]);
+      echo "</td>";
+      $randDropdown = mt_rand();
+      echo "<td><label for='dropdown_manufacturers_id$randDropdown'>".__('Manufacturer')."</label></td>";
+      echo "<td>";
```

Comment:
an Appliance is a group of different items with there own manufacturer.
So this field is unneeded

Reply:
Appliance feature in GLPI is a mix beetween Appliances and Webapplications plugins; this field was present in Webapplications. Also, it seems it should make sense on some cases, so since this is not a mandatory field, it can stay as is.